rollingpin/args.py

```diff
@@ -110,6 +110,15 @@ def _add_flags(config, parser):
         dest="list_hosts",
     )
 
+    options_group.add_argument(
+        "--dangerously-fast",
+        action="store_true",
+        default=False,
+        help="Don't wait on service restarts. \
```

Comment:
it's cleaner to use automatic string concatenation than escaping the newline, see the other long descriptions in this file for comprehensive examples. basic idea is just :

```python
long_string = ("the beginning"
    "the middle"
    "the end"
)
```

automatically turns into one string.